src/site/xdoc/cmdline.xml.vm

```diff
@@ -1072,6 +1092,13 @@ class Test {
 &lt;/suppressions&gt;
   </code></pre>
 </div>
+  <div class="wrapper">
+  <p>The XML file generated is intended to be used with the
+    <a href="../filters/suppressionfilter.html">SuppressionFilter</a>
+    or
+    <a href="../filters/suppressionsinglefilter.html">SuppressionSingleFilter</a>
```

Comment:
Better to remove this OR and filter .
https://checkstyle.org/filters/suppressionsinglefilter.html#Example1-config
Has different syntax, better to not mention it here at all